### PR TITLE
Jsonize bash and cut protection from bionics

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Infrastructure "Jsonize bash and cut protection from bionics"```

#### Purpose of change
Jsonize bash and cut protection from bionics

#### Describe the solution
Add bas_protect and cut_protect to bioinics

#### Testing
image

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
I wanna see it generalized for all damage types eventually, but that's a good start.

File: data/json/bionics.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "name": "Alloy Plating - Arms",
     "description": "The flesh on your arms has been surgically replaced by alloy plating.  Provides passive protection and can be used in conjunction with bionic martial arts.",
     "occupied_bodyparts": [ [ "ARM_L", 4 ], [ "ARM_R", 4 ] ],
+    "bash_protec": [ [ "ARM_L", 3 ], [ "ARM_R", 3 ] ],
```
Comment (Kelenius) [inline data/json/bionics.json]: `bash_protec`? `bash_protect`?

Comment (ZhilkinSerg) [reply data/json/bionics.json]: That's for consistency with existing `env_protec`.

Comment (Fris0uman) [reply data/json/bionics.json]: `bash_protec` as in `env_protec` and `cut_protec`

**Pull request merged**